### PR TITLE
Api 16995 - 2 New Letter Gen Test Users

### PR DESCRIPTION
**Pull request by aaronhietpas**
Add BGS responses for 2 new mock users to generate the new Benefit Summary Dependent letter. This data is pulled from other mock users and modified for the new letter.

### CODE REVIEW

**Comment by dbrown-va** [comment]
Earlier test user info was included in this table.

https://github.com/department-of-veterans-affairs/lighthouse-mock-benefits-gateway-services/blob/master/mock-bgs/README.md

Recent PR's have not kept up with this documentation, but it may be helpful to keep this up to date.  Or possibly even a script that generates this based on the mockdata.

File: mock-bgs/src/main/resources/data/claimant/general-info-response/100000000.xml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    <numberOfFlashes>6</numberOfFlashes>
+    <nursingHomeIndicator>N</nursingHomeIndicator>
+    <nursingHomeName/>
+    <paidThroughDate>04302022</paidThroughDate>
+    <paraplegicHousingNumber/>
+    <payStatusTypeCode>AUTH</payStatusTypeCode>
+    <payStatusTypeName>Authorized</payStatusTypeName>
+    <payeeBirthDate>06151935</payeeBirthDate>
+    <payeeName>MARSHALL ROGER</payeeName>
+    <payeeSSN>796127023</payeeSSN>
```
Comment (dbrown-va) [inline mock-bgs/src/main/resources/data/claimant/general-info-response/100000000.xml:77]: This does start to diverge from the mock MPI record proposal.

https://github.com/department-of-veterans-affairs/lighthouse-mock-mpi/pull/39/commits/942a7b5534fbce3c93f57ab076b576da97763f61#diff-2b668c82da4663dc8dee36a6d864a2d85033cc8b03fe262072d95963ca6ebe75R70

Comment (aaronhietpas) [reply mock-bgs/src/main/resources/data/claimant/general-info-response/100000000.xml]: Updated SSN values in files for both users

**Comment by aaronhietpas** [comment]
> Earlier test user info was included in this table.
> 
> https://github.com/department-of-veterans-affairs/lighthouse-mock-benefits-gateway-services/blob/master/mock-bgs/README.md
> 
> Recent PR's have not kept up with this documentation, but it may be helpful to keep this up to date. Or possibly even a script that generates this based on the mockdata.

How do we come up with/assign the emails to a user? 
For Letter Generator the ICN is passed directly in the rest call so we don't need to have that set up.

**Comment by dbrown-va** [comment]
> > Earlier test user info was included in this table.
> > https://github.com/department-of-veterans-affairs/lighthouse-mock-benefits-gateway-services/blob/master/mock-bgs/README.md
> > Recent PR's have not kept up with this documentation, but it may be helpful to keep this up to date. Or possibly even a script that generates this based on the mockdata.
> 
> How do we come up with/assign the emails to a user? For Letter Generator the ICN is passed directly in the rest call so we don't need to have that set up.

That is a good point.  Really that table should be ICN/SSN based, and not tie to an email.

The OAuth email/password is a separate component.  This is likely outside the scope of your current story, so I will go ahead and approve.

**Review by dbrown-va — APPROVED** [review]

**Pull request merged**